bin/regen.sh

```diff
@@ -41,7 +41,7 @@ $MYSQLCMD < civicrm_data.mysql
 $MYSQLCMD < civicrm_sample.mysql
 echo "DROP TABLE IF EXISTS zipcodes" | $MYSQLCMD
 $MYSQLCMD < zipcodes.mysql
-php GenerateData.php
+env CIVICRM_UF=UnitTests php GenerateData.php
```

Comment:
On a second read, this might be too good to be true -- the UnitTests may be connecting to a different MySQL DB (at least, different from the one used to dump the final `civicrm_generated.mysql`). It bears a closer look.